### DISCUSSION TITLE
[Project beta] Logical filter between different fields

### DISCUSSION THREAD

**Discussion by jvanz** [community/community#12418] [Projects and Issues]
I work in a project where we contribute on several repositories. Some of them are manager by our team. Other are Open Source projects which we contribute to. But we keep track of our work in Github boards (old and new version).

I would like to aggregate in a board view all the changes related to the same "milestone" or "related changes" from different repositories (managed by the team or not). For that I need filter which allows me more complex expressions like: "I want issue with label x OR issues from these repositories". Maybe something like this: `label:kubernetes or repo:kubernetes/kubernetes`

The "or" operation are allowed only for values in the same field. Which does not work in our situation because we do not have power to assign label for repositories outside our organization. Besides that, even if we could set the labels. Our label may not be meaningful in the Open Source project repository.

**Comment by jasononeil** [comment]
A less complicated filter where this would still be useful:

`assignee:@me or reviewer:@me`

Currently there's no way to get a view of things I'm assigned to work on _or_ assigned to review, as far as I can tell. We're looking at using a [Github Issue Notebook](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-github-issue-notebooks) view to work around it, but would prefer to have a view like this saved on our project board.
